lib/consts/Types.ts

```diff
@@ -14,6 +14,8 @@ export type SwapUpdate = {
 export type CurrencyConfig = {
   symbol: string;
 
+  timeoutBlockNumber: number;
```

Comment:
Let's call this variable `timeoutBlockDelta` instead. That might be less confusing and irritating